client/my-sites/sharing/connections/service-action.jsx

```diff
@@ -83,10 +83,22 @@ const SharingServiceAction = ( {
 	if ( primary && 'mailchimp' === service.ID ) {
 		return (
 			<div>
-				<Button className="connections__signup" compact>
+				<Button
+					className="connections__signup"
+					compact
+					href="https://public-api.wordpress.com/rest/v1.1/sharing/mailchimp/signup"
+					target="_blank"
```

Comment:
Just making sure this button is returning `rel="noopener noreferrer"` since those are needed for external _blank targeted links?

Reply:
Button is smart like that https://github.com/Automattic/wp-calypso/blob/master/client/components/button/index.jsx#L40

Reply:
Excellent, thanks!